tools/npz2rds/test/test.sh~

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
I assume this is a cache file. Could you remove this from the PR?

Reply:
removed